### ISSUE TITLE
`normalize` can't handle NaNs

### ISSUE THREAD

**Issue by gottacatchenall** [EcoJulia/NeutralLandscapes.jl#61]
**Describe the bug**
If you call `normalize` on an initial landscape with NaNs, some numbers get turned into nans

**To Reproduce**

~~~ juliausing NeutralLandscapes
using Plots
using SimpleSDMLayers
using Distributions

ENV["SDMLAYERS_PATH"] = "/home/michael/data/"
quebec = SimpleSDMPredictor(WorldClim, BioClim; left=-90., right=-55., top=60., bottom=45.)
qcmask = fill(true, size(quebec)) 
qcmask[findall(isnothing, quebec.grid)] .= false

init = rand(PerlinNoise((2,2)), size(quebec), mask=qcmask)

svu = SpatiotemporallyAutocorrelatedUpdater(
    rate = 0.01,
    variability = 0.01,
    spatialupdater = PlanarGradient(90)
)

seq = normalize(update(svu, init, 30))
~~~

**Expected behavior**
Should leave NaNs in place but not convert numbers to NaNs

**Comment by mkborregaard** [comment]
Probably `findmin` and `findmax`? Consider replacing with `NaNMath.min`?

**Comment by gottacatchenall** [comment]
> Probably findmin and findmax? Consider replacing with NaNMath.min? 

Yes I think so, should be an easy fix

**Comment by gottacatchenall** [comment]
closed in #62
